lib/rubocop/cop/lint/useless_access_modifier.rb

```diff
@@ -152,7 +160,18 @@ def start_of_new_scope?(child)
 
         def instance_eval_or_new_call?(child)
```

Comment:
Hmm, this method name isn't really correct any more (it probably wasn't quite right when I created it!). Since all these conditions are now essentially "do we end up calling `{class,module,instance}_eval`" I wonder if a better name is simply `eval_call?(child)`